src/store/state/resource-model.ts

```diff
@@ -1,5 +1,24 @@
 import { ResourceIF } from '@/interfaces'
 
 export const resourceModel: ResourceIF = {
-  certifyStatementResource: null
+  entityType: '',
```

Comment:
I know '' is falsey just as null is for a string, but do you think we should assign null to these empty strings to really denote that they have no value?

Reply:
Totally, makes sense!

Reply:
Updated 👍